### ISSUE TITLE
[automation] updatenodes shows "Error: Unsupported request" for almost all options in rhels6.9 and sles11.4

### ISSUE THREAD

**Issue by hu-weihua** [xcat2/xcat-core#4517]
xcat version:
Version 2.13.9 (git commit af7bfe8489888dae3ef285233d1edc468750b051, built Wed Dec 13 06:16:04 EST 2017)

``updatenodes`` shows ``Error: Unsupported request`` for almost all options in rhels6.9 and sles11.4. This result in almost all related cases failed. 

For example:
```
RUN:updatenode c910f02c02p28 -P confignetwork [Wed Dec 13 10:37:17 2017]
ElapsedTime:1 sec
RETURN rc = 0
OUTPUT:
Error: Unsupported request
CHECK:rc != 0   [Failed]
```

**Comment by whowutwut** [comment]
Are we doing something that is not supported on the older OS? RHEL 6.x and SLES 11.x

**Comment by immarvin** [comment]
@whowutwut , the root cause has been found, this is caused by a PR merged yesterday,

**Comment by robin2008** [comment]
The error is caused by using array ref in updatenode, the usage will cause wrong in old perl version. And this is introduced by PR 4484.

Now we fix it by PR#4518
